### ISSUE TITLE
Allow the use of lodash functions and variables

### ISSUE THREAD

**Issue by shoulders** [quick-lint/quick-lint-js#1243]
Lodash is a popular Javascript Library and it has a particular way of denoting its variables and functinos (there might be other things) and I would like these not to be flagged as errors.

in particular the use of `_`  `_(` `_.`

Currently the following are flagged as errors, but there are not and they work as expected in my code.

**Function**
```javascript
_(data).map(function(data){}
```
![Image](https://github.com/user-attachments/assets/fc688e9a-42da-4182-b4b1-5a481e30c3ac)

**Variable**
```javascript
_.map(timeScaleRanges, "label")
```
![Image](https://github.com/user-attachments/assets/3a878dc0-337e-49a4-9b3c-b02576b0f94b)

### Links
- https://lodash.com/
